src/pages/dar_application/ResearcherInfo_new.js

```diff
@@ -0,0 +1,641 @@
+                    className: 'checkbox-inline rp-checkbox',
+                    disabled: !isNil(darCode),
+                    required: true,
+                    checked: localUse,
+                    onChange: (e) => formFieldChange({name: 'localUse', value: e.target.checked})
+                  }),
+                  label({ className: 'regular-checkbox rp-choice-questions', htmlFor: 'local-requested' },
+                    ['I am requesting permission to use local computing to carry out the research described in my Research Use Statement']
+                  )
+                ])
+              ]),
+            ]),
+            div({className: 'row no-margin', isRendered: cloudUse === true}, [
+              div({className: 'col-lg-6 col-md-6 col-sm-12 col-xs-12 rp-group'}, [
+                h(FormField, {
+                  id: 'cloudProvider',
+                  title: 'Name of Cloud Provider',
+                  onChange: ({ key: name, value }) => formFieldChange(name, value),
+                  defaultValue: cloudProvider,
+                  validators: [FormValidators.REQUIRED],
+                  disabled: !isEmpty(darCode)
+                })
+              ]),
+              div({className: 'col-lg-6 col-md-6 col-sm-12 col-xs-12 rp-group'}, [
+                h(FormField, {
+                  id: 'cloudProviderType',
+                  title: 'Type of Cloud Provider',
+                  defaultValue: cloudProviderType,
+                  validators: [FormValidators.REQUIRED],
+                  disabled: !isNil(darCode),
+                  onChange: ({ key: name, value }) => formFieldChange(name, value)
+                })
+              ])
+            ]),
+            div({className: 'row no-margin', isRendered: cloudUse === true}, [
+              div({className: 'col-lg-12 col-md-12 col-sm-12 col-xs-12 rp-group'}, [
+                textarea({
+                  style: {
+                    backgroundColor: cloudInputStyle(cloudProviderDescription).backgroundColor,
+                    width: '100%',
+                    padding: '1rem'
+                  },
+                  defaultValue: cloudProviderDescription,
+                  disabled: !isNil(darCode),
+                  onBlur: (e) => formFieldChange({name: 'cloudProviderDescription', value: e.target.value}),
+                  name: 'cloudProviderDescription',
+                  id: 'cloudProviderDescription',
+                  rows: '6',
+                  required: true,
+                  placeholder: 'Please describe the type(s) of cloud computing service(s) you wish to obtain (e.g PaaS, SaaS, IaaS, DaaS)'
+                    + ' and how you plan to use it (them) to carry out the work described in your Research Use Statement (e.g. datasets to be included, process for data transfer)'
+                    + ' analysis, storage, and tools and/or software to be used. Please limit your statement to 2000 characters',
+                  maxLength: 2000
```

Comment:
migrate this to a formfield?

Reply:
Added a new Textarea formfield component!